static/src/javascripts/projects/common/modules/discussion/comments.js

```diff
@@ -0,0 +1,606 @@
+import Component from 'common/modules/component';
+import DiscussionApi from 'common/modules/discussion/api';
+import CommentBox from 'common/modules/discussion/comment-box';
+import WholeDiscussion from 'common/modules/discussion/whole-discussion';
+import relativedates from 'common/modules/ui/relativedates';
+import userPrefs from 'common/modules/user-prefs';
+import { inlineSvg } from 'common/views/svgs';
+
+type userType = {
+    badge: Array<{
+        name: string,
+    }>,
+    displayName: string,
+    isStaff?: boolean,
+    privateFields: {
+        canPostComment: boolean,
+    },
+};
+
+type commentType = {
+    id: string,
+    body: string,
+};
+
+const PREF_RELATIVE_TIMESTAMPS: string = 'discussion.enableRelativeTimestamps';
+
+const shouldMakeTimestampsRelative = (): boolean =>
+    userPrefs.get(PREF_RELATIVE_TIMESTAMPS) !== null
+        ? userPrefs.get(PREF_RELATIVE_TIMESTAMPS)
+        : true;
+
+class Comments extends Component {
+    constructor(options: Object): void {
+        super();
+
+        this.componentClass = 'd-comments';
+        this.comments = null;
+        this.topLevelComments = null;
+        this.user = null;
+        this.classes = {
+            comments: 'd-thread--top-level',
+            topLevelComment: 'd-comment--top-level',
+            reply: 'd-comment--response',
+            showReplies: 'd-show-more-replies',
+            showRepliesButton: 'd-show-more-replies__button',
+            newComments: 'js-new-comments',
+            comment: 'd-comment',
+            commentReply: 'd-comment__action--reply',
+            commentPick: 'd-comment__action--pick',
+            commentStaff: 'd-comment--staff',
+            commentBody: 'd-comment__body',
+            commentTimestampJs: 'js-timestamp',
+            commentReport: 'js-report-comment',
+        };
+        this.defaultOptions = {
+            discussionId: null,
+            showRepliesCount: 3,
+            commentId: null,
+            order: 'newest',
+            threading: 'collapsed',
+        };
+
+        this.setOptions(options);
+    }
+
+    getMoreReplies(event: Event) {
+        event.preventDefault();
+
+        const currentTarget: HTMLElement = (event.currentTarget: any);
+
+        const li: HTMLElement = ($.ancestor(
+            currentTarget,
+            this.getClass('showReplies').slice(1)
+        ): any);
+
+        if (li) {
+            li.innerHTML = 'Loading…';
+        }
+
+        const source = bonzo(event.target).data('source-comment');
+        const commentId = currentTarget.getAttribute('data-comment-id');
+
+        if (commentId) {
+            const endpoint = `/discussion/comment/${commentId}.json?displayThreaded=true`;
+
+            fetchJson(endpoint, {
+                mode: 'cors',
+            })
+                .then(resp => {
+                    const comment = bonzo.create(resp.html);
+                    let replies = qwery(this.getClass('reply'), comment);
```

Comment:
`const replies = qwery(this.getClass('reply'), comment).slice(this.options.showRepliesCount);`?